bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/secondgeneration/SecondGenerationHandler.java

```diff
@@ -327,7 +334,7 @@ private void refresh() {
             }
         } catch (final RuntimeException e) {
```

Comment:
In fact, `refresh` is called only by the scheduled job by initialize.
You have to choose where do you catch `RuntimeException`; doing it at both places has no real sense.
I would suggest to remove the exception handling here.